R/utils.R

```diff
@@ -1,9 +1,77 @@
+#' @rdname utils
+#' @title Utility Functions
+#'
+#' @description Utility functions useful for package default behavior.
+#' `pkg_test` returns the path to tests from the `dbtest` package.
+#' `pkg_config` returns the path to connection config from the `dbtest`
+#' package.  `all_tests` returns the path to all .yml or .yaml files in a given
+#' directory.  The default is to return all tests in the `dbtest` package
+#'
+#' @param file optional The file to reference at the default location
+#' @param dir optional The directory for which to show all_tests
+#'
 #' @export
-default_tests_path <- function(file = "tests.yml"){
-  system.file("extdata", file, package = "dbtest")
+pkg_test <- function(file = "simple-tests.yml"){
```

Comment:
I'm afraid that adding a "simple" prefix to the name of the file may give the impression that is not a complete set of tests

Reply:
:+1: See below on multiple YAML files and the `all_tests` function in `utils.R`